pkgs/by-name/co/comaps/package.nix

```
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  fetchFromGitHub,
+  fetchFromGitea,
+  fetchpatch,
+
+  cmake,
+  git,
+  qt6,
+  python3,
+  boost,
+
+  xorg,
+}:
+let
+  mapRev = 250602;
+
+  worldMap = fetchurl {
+    url = "https://cdn.comaps.app/maps/${toString mapRev}/World.mwm";
+    hash = "sha256-VhlY7XDvNsyP9dfAvprpjvHoIjmS82XNbCIFjlto1Ng=";
+  };
+
+  worldCoasts = fetchurl {
+    url = "https://cdn.comaps.app/maps/${toString mapRev}/WorldCoasts.mwm";
+    hash = "sha256-nlkeQpYje5LbD6exB1L72jYJnXEKBKho6sYgPxgorR8=";
+  };
+
+  worldFeedIntegrationTestsData = fetchFromGitHub {
+    owner = "organicmaps";
+    repo = "world_feed_integration_tests_data";
+    rev = "30ecb0b3fe694a582edfacc2a7425b6f01f9fec6";
+    hash = "sha256-1FF658OhKg8a5kKX/7TVmsxZ9amimn4lB6bX9i7pnI4=";
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "comaps";
+  version = "2025.06.22-5";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "comaps";
+    repo = "comaps";
+    tag = "${finalAttrs.version}-android";
+    hash = "sha256-InYrOXpQaen6Th8jlmpVwMkf5G7knK7J781CRt5Jvz4=";
+    fetchSubmodules = true;
+  };
+
+  patches = [
+    # The world feed integration data is fetched externally, instead we fetch it ourselves.
+    ./local-world-feed-integration-tests-data.patch
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    git
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qtpositioning
+    qt6.qtsvg
+    python3
+
+    boost
+  ] ++ (with xorg; [
+    libXrandr
+    libXinerama
+    libXcursor
+  ]);
```

```suggestion
    libXrandr
    libXinerama
    libXcursor
  ];
```
X11 libs are available at top level inside `callPackage`d, and, judging by recent efforts, `xorg` namespace seems to get deprecated and dissolved